decs.d.ts

```diff
@@ -84,7 +107,7 @@ declare module '@gridsuite/commons-ui' {
             position: string;
             text: string;
         };
-        customAdornment?: ReactElement;
+        customAdornment?: ReactElement | null;
```

Comment:
it can be null
initial state is null. in addition, it can be set to null.
src/components/dialogs/commons/use-name-check.tsx

Reply:
alright, it might have worked with undefined as well

Reply:
if we shoose undefined, type have to be 
ReactElement | undefined.
because, in the case that we have not an adornment, we have to set the customAdornement value to null or undefined